misc/chip_data.json

```diff
@@ -693,7 +693,7 @@
     },
     "aaa4e58f-16df-426a-b7a7-a654eab97037": {
       "ReadonlyName": "Asin",
-      "Description": "",
+      "Description": "this node allows you to use the math formula Asin",
```

Comment:
let's use the word "chip" instead of node where possible. we know people will call things nodes but we want to be consistent about official terminology

Reply:
Done!